blog-service/2023-08-09-metrics-slos.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Metrics-Based SLOs (Reliability Management)
+image: https://www.sumologic.com/img/logo.svg
+keywords:
+  - slo
+  - observability
+  - metrics
+  - operators
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+You can now create SLOs from the **Metrics** page and use [metrics operators](/docs/metrics/metrics-operators) in these SLO queries. The metrics query specified in your SLO should be a pure selector query. You can specify one or more operators in the query for SLO.
```

Comment:
`You can now create SLOs from the **Metrics** page` - this is already supported. This change is for supporting operators in metrics-based SLOs.

Reply:
`The metrics query specified in your SLO should be a pure selector query.` - This is in conflict with this change and not correct.

Reply:
@tarunk2 @jinkane ,  thank you both for the clarification! Can you please provide what should be the correct edits in suggest mode? Then I'll accept the comments and we can publish.

https://github.com/SumoLogic/sumologic-documentation/assets/56411016/d0213a22-aef8-4a93-aeae-c885979f5b8e